### PR TITLE
Setup controllers

### PR DESCRIPTION
**Pull request by zieeco**
**In this milestone, I did the following**

- Added the `users` and `posts` controllers
- Added  the views for the controllers
- Added the routes in the `routes.rb` file

This part of the blog app features the setting up of the controllers for the app.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def index; end
+
+  def show; end
```
Comment (Davidosky007) [inline app/controllers/posts_controller.rb:4]: * As stated in the [project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#:~:text=Make%20sure%20that%20your%20app%20handles%20the%20URLs%20used%20in%20the%20wireframes) `Make sure that your app handles the URLs used in the wireframes` and example screenshot below ⬇️, Kindly fix this as when i visit the URL it gives me errors 
image

File: config/routes.rb [hunk 2]
```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
+  get 'posts/posts'
+  get 'home/home'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
+  resources :users, only: %w[:index :show] do
+    resources :posts, only: %w[:index :show]
```
Comment (Davidosky007) [inline config/routes.rb]: * As stated in the project requirement `Make sure that you added code to handle only the 4 required URLs.` , Kindly make sure these URLs work as expected, as when i run `rails routes` i get a different URL. below are screenshots ⬇️ 
image
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @zieeco 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Good readme file 👍 
* Well written PR description 👍 
* Proper use of Gitflow 👍 
*  Nice commit messages 👍 

## Required Changes ♻️
* Kindly work on your routes and views pages to suit the URLs provided in the wireframes as required below are the highlighted section of the rpoject requitrements still not implemented 
image
 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (zieeco) [reply app/controllers/posts_controller.rb:4]: - [ ] This issue has been resolved :heavy_check_mark:

File: config/routes.rb (hunk 2 above)
Comment (zieeco) [reply config/routes.rb]: - [ ] This issue has been resolved :heavy_check_mark:

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @zieeco👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**